### DISCUSSION TITLE
SNN app vs web app

### DISCUSSION THREAD

**Discussion by CyberSmurfar** [standardnotes/forum#3373] [General]
Hi

I have downloaded the SN windows app but are there any differencies between the Win app and Web app or are they exactly the same with same features?

**Comment by effieeee** [comment]
A notable feature found only on the desktop apps is the ability to automatically backup your notes and files to a chosen location in your device:
![image](https://github.com/standardnotes/forum/assets/54989748/0f0b0c6d-aa08-4bf2-a145-37877efec54a)
